src/routes/posts.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+
+const db = require('../db')
+
+const router = express.Router()
+
+// CREATE AN ARTICLE
+
+const createPosts = async (req, res, next) => {
+    try {
+        const { id, userId, title, image, content, published, createdAt } =
```

Comment:
@beblicarl not sure if maybe there are more changes yet to be done but just to clarify it seems we're still getting it from the`req.body` though and inserting it into the db. these are autogenerated fields by the database itself so should not be gotten from the request

 we should insert without the id and createdAt and after the insert use the response from the db to get the id and created at

Reply:
alright

Reply:
also, how do i send id and createdAt in the response body if I don't get it from the request body ? @jocrah 
![image](https://user-images.githubusercontent.com/68056104/185152555-356fcb63-5f98-428b-9eda-bf96bd54dde0.png)